components/Icon/index.jsx

```diff
@@ -0,0 +1,11 @@
+import Image from 'next/image';
+import PropTypes from 'prop-types';
+
+const Icon = ({ src, alt, ...attributes }) => <Image src={src} alt={alt} {...attributes} />;
+
+Icon.propTypes = {
+  src: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+  alt: PropTypes.string.isRequired,
+};
+
+export default Icon;
```

Comment:
As its usage is not restricted to `icons`, should we name it "Image" instead? 

You might need to play with aliases, something like:

```
import NextImage from 'next/image';
```

Reply:
sure, done in e9f3815051be75cb8f3734494b6d05c9fc389acf